tools/peptideshaker/searchgui.xml

```diff
@@ -330,19 +887,25 @@
             <param name="variable_modifications" value="oxidation of m"/>
             <param name="min_charge" value="1"/>
             <param name="max_charge" value="3"/>
-            <param name="advanced_type_selector" value="advanced"/>
-            <param name="xtandem_selector" value="yes"/>
-            <param name="xtandem_selector.xtandem_refine_selector" value="yes"/>
-            <param name="omssa_selector" value="no"/>
-            <param name="msgf_selector" value="yes"/>
-            <param name="ms_amanda_selector" value="no"/>
+            <param name="XTANDEM_selector" value="True"/>
+            <param name="xtandem.xtandem_advanced" value="yes"/>
+            <param name="xtandem_advanced.xtandem_refine_selector" value="yes"/>
+            <param name="OMSSA_selector" value="False"/>
+            <param name="MSGF_selector" value="True"/>
+            <param name="TIDE_selector" value="False"/>
+            <param name="MYRIMATCH_selector" value="False"/>
```

Comment:
Totally agree regarding adding tests.  The main problem now is that I cannot get automated tests to work.  Seems to have been some testing upheavals in galaxy lately that have broken the way I used to do it with planemo.